rust/optify/src/builder.rs

```diff
@@ -141,6 +142,13 @@ fn resolve_imports(
     Ok(())
 }
 
+type LoadingResult = (
```

Comment:
Maybe this should be a `struct`?

Reply:
I think some rust-analyzer thing made this automatically, but yeah I'll change it.